src/directives/automation/editor/panel/dpEditorPanelWhatsapp.js

```diff
@@ -265,6 +300,62 @@
         }
       }
 
+      function uploadImageSelect(e) {
```

Comment:
this function should be named something like: `uploadFileSelect`, right? I mean, it refers to a file, instead an image.

Reply:
rename done, nice suggest